EPMEDU-752: Align "More" page with new design

Fileek opened the pull request.

Fileek committed:
[EPMEDU-752]: Align "More" page with new design

Fileek commented:
https://user-images.githubusercontent.com/83027107/176613302-7d38b675-bd54-4323-bced-14ca752e8890.mp4

egorikftp commented on base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt:
```diff
@@ -0,0 +1,46 @@
+fun ShortButtonPreview() {
+    AnimealTheme {
+        Surface {
+            ShortButton(text = "Click", onClick = {})
```
please add preview for disabled state

Fileek replied:
Done

Fileek committed:
Sort screens in packages,
add part of ProfileScreen.kt design,
replace MainViewModel.kt initialization and
change TopBar.kt and ShortButton.kt implementations

Fileek commented:
https://user-images.githubusercontent.com/83027107/176650922-809c8e85-e306-4ff1-b4a6-756e250c6b1f.mp4

egorikftp commented on base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt:
```diff
@@ -0,0 +1,56 @@
+fun ShortButton(
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
```
according to google guidlines modifier should be a first parameter

Fileek replied:
Sorry, but documentation says: "It should appear as the first optional parameter in the parameter list; **after all required parameters** (except for trailing lambda parameters) but before any other parameters with default values.",
text and onClick parameters are required, therefore modifier should be after them.

Fileek committed:
Rearrange and rename classes and composables

Fileek committed:
Fix detekt issue
